tests/conftest.py

```diff
@@ -914,7 +914,8 @@ def health_checker(request, tier_marks_name):
     def finalizer():
         try:
             teardown = config.RUN['cli_params']['teardown']
-            if not teardown:
+            skip_ocs_deployment = config.ENV_DATA['skip_ocs_deployment']
```

Comment:
```suggestion
            skip_ocs_deployment = config.ENV_DATA.get('skip_ocs_deployment', False)
```

Reply:
I double checked the default config and seems we have those default values in the config so in this case I will not block this PR as it's probably can block engineering as they have separated pipeline of OCP/OCS deployment. So I will go ahead and merge this one as I see the verification job passed.  But anyway it will be better to change it to get method as this will not rely on this config value if not specified.

Reply:
Deploy job passed with skip_ocs_deployment: https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/6660/consoleFull

By default, skip_ocs_deployment will be False. Even for engineering I believe all the default values ( default_config.yaml ) will be loaded.